app/views/static_page/help.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= provide :title, "help" %>
+
+<h1>Help</h1>
+<p>
+	
+Get help on the Ruby on Rails Tutorial at the
+<a href="http://www.railstutorial.org/help">Rails Tutorial
+help page</a>.
+
+To get help on this sample app, see the
+<a href="http://www.railstutorial.org/book"><em>Ruby on Rails
+Tutorial</em>
+book</a>.
+
+</p>
```

Comment:
Fix this error

Reply:
please add end line (blank line) for each file

Reply:
,which kind of file?

Reply:
I try to edit but I not sure am I doing right